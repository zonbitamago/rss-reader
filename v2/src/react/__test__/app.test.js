'user strict';
jest.dontMock('../components/app.jsx');
import React from 'react';
import renderer from 'react-test-renderer';
import App from '../components/app.jsx';
import mockdate from 'mockdate';

test('sample', function() {
  expect(1).toBe(1);
});

test('snapshot', function() {
  mockdate.set(1434319925275);
  const tree = renderer.create(<App/>).toJSON();
  expect(tree).toMatchSnapshot();

  mockdate.reset();
});
