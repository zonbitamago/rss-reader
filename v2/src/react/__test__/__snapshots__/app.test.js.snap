// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<main>
  Hello World!!
  <aside>
    side
  </aside>
  <section>
    mainPanels
  </section>
</main>
`;
